internal/cli/root.go

```diff
@@ -62,6 +62,17 @@ func NewRootCommand(s state.State) *cobra.Command {
 		primaryip.NewCommand(s),
 	)
 	cmd.PersistentFlags().Duration("poll-interval", 500*time.Millisecond, "Interval at which to poll information, for example action progress")
+	cmd.PersistentFlags().Bool("quiet", false, "Only print error messages")
 	cmd.SetOut(os.Stdout)
```

Comment:
Could you move this inside to function below, so the logic is self-contained.

Reply:
Done in 32afd2fbca6c1b0358a58e5d2c56601b27a11a39